src/components/MetricsPane/charts/TimeSeriesFishBiomass.jsx

```diff
@@ -89,6 +89,11 @@ export const TimeSeriesFishBiomass = () => {
     hovertemplate: `${rule}<br>Year: %{x}<br>%{y:.0f} kg/ha<extra></extra>`,
   }))
 
+  const allSingleYear = plotlyDataConfiguration.every((data) => {
+    const uniqueYears = Array.from(new Set(data.x))
```

Comment:
Question: Does this need to be converted back to an array after converting it to a Set? Could you use something like
```suggestion
    const uniqueYears = new Set(data.x)
    return uniqueYears.size === 1
```